source/server/data_moniker_service.cpp

```diff
@@ -103,20 +111,31 @@ void DataMonikerService::InitiateMonikerList(const MonikerList& monikers, Endpoi
   }
 }
 
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+#ifndef _WIN32
+void set_moniker_stream_processor(int stream_processor)
```

Comment:
In the context of this method we are just setting the cpu affinity, irrespective of who would be calling be it moniker or any other code. We could name is set_cpu_affinity and take in the thread_id as well

Reply:
>the main purpose of this helper is thus to assign a specific processor to a thread for moniker streaming apis.
Hence the name, set_moniker_stream_processor sounds correct to me.

The API (or rather the private helper) doesn't have anything to do with moniker. Update name to `set_cpu_affinity` and change parameter name to `cpu`. The parameter as well has nothing to do with "stream".

Reply:
Updated